woocommerce/woocommerce
# Make cys intro page responsive

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Mrk6SERPZ4KrFHSjM0a8TK-fi-6179%3A40430

Make cys intro page responsive for large screens

\>= 1600px
<img width="1600" alt="Intro screen" src="https://github.com/woocommerce/woocommerce/assets/4344253/5f8dc8e0-7ee7-441a-9ed4-8118e4d260d7">

< 1600px
<img width="1550" alt="Intro screen (1)" src="https://github.com/woocommerce/woocommerce/assets/4344253/abb3d604-23e9-4cfa-9739-f2958238c613">




### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new WooCommerce installation with this version.
2. Make sure to enable customize-store feature flag
3. Go to WooCommerce -> Home and click Customize Store task
4.  Adjust the window size and make sure the page is displayed as designed



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
